core/src/main/java/greencity/controller/ClientController.java

```diff
@@ -273,4 +273,26 @@ public ClientController(UBSClientService ubsClientService) {
         return ResponseEntity.status(HttpStatus.OK)
             .body(ubsClientService.getOrderInfoForSurcharge(orderId, languageId));
     }
+
+    /**
+     * Controller returns bonus points of current user.
+     * {@link greencity.dto.UserVO}.
+     *
+     * @param userUuid {@link String} uuid.
+     * @return {@link UserPointDto}.
+     * @author Max Boiarchuk
+     */
+    @ApiOperation(value = "Get current user points.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = UserPointDto.class),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @GetMapping("/get-user-bonuses")
```

Comment:
no need to name it "/get-user-bonuses" as it is method GET and it is obvious. name it just /user-bonuses